src/dashboard/src/components/administration/views.py

```diff
@@ -88,14 +89,40 @@ def failure_report_detail(request):
     return render(request, 'administration/reports/failure_report_detail.html', locals())
 
 def atom_dips(request):
-    upload_setting = models.StandardTaskConfig.objects.get(execute="upload-qubit_v0.0")
-    form = AtomSettingsForm(request.POST or None, instance=upload_setting)
+    """ View to configure AtoM DIP upload. """
+    initial_data = _intial_settings_data()
+    form = AtomDipUploadSettingsForm(request.POST or None, prefix='storage',
+        initial=initial_data)
     if form.is_valid():
+        # Produce a set of commandline arguments for the AtoM upload job
+        upload_setting = models.StandardTaskConfig.objects.get(execute="upload-qubit_v0.0")
+        opts = []
+        char_fields = ['dip_upload_atom_url', 'dip_upload_atom_email',
+                       'dip_upload_atom_password', 'dip_upload_atom_rsync_target',
+                       'dip_upload_atom_rsync_command', 'dip_upload_atom_version']
+        for field in char_fields:
+            value = form.cleaned_data.get(field)
+            if not value:
+                continue
+            optname = field.replace('dip_upload_atom_', '').replace('_', '-')
+            opts.append('--{}={}'.format(optname, value))
```

Comment:
Put quotes around the value passed as a parameter?  `'--{}="{}"'.format(optname, value)`